src/main/java/com/elastic/support/BaseService.java

```diff
@@ -77,9 +76,8 @@ public void createArchive(String tempDir) {
             archiveUtils.createArchive(tempDir, archiveFilename);
 
         } catch (Exception ioe) {
-            logger.error("Couldn't create archive. {}", ioe);
+            logger.info("Couldn't create archive. {}", ioe);
```

Comment:
Not that I _really_ care, but this seems like a solid reason for an error versus info?

Reply:
If you go back and check the convention, all of the log statements that aren't logger.log are info. That's because I use a custom appender level that writes everything to to the bundled log file and only some things to the console. None of which even display the level now. So what level you log it at is lost on whoever's looking at the console. Until the appender gets configured though, i still need some default configured behavior to output messages. It's easy to get one level, a lot more work to get more than that. With no visible payoff.